R/01-webdata-obj.R

```diff
@@ -108,7 +115,11 @@ setMethod("webdata", signature("character"), function(.Object=c("prism",  "iclus
 setMethod("webdata", signature("geojob"), function(.Object, ...) {
   xmlVals <- inputs(xmlParse(xml(.Object)))
   url <- xmlVals[["DATASET_URI"]]
-  times <- c(xmlVals[["TIME_START"]], xmlVals[["TIME_END"]])
+  if(any(grepl(pattern = "TIME_",  x = names(xmlVals)))){ #underscore is important
```

Comment:
I think this also needs to deal with the case where there is a START but no END, or an END with now START.